### PR TITLE
Added pressure capabilities from sensors

### PR DESCRIPTION
**Pull request by Phara0h**
Also put in a PR to hubitatmaker for the adding pressure as well.

### CODE REVIEW

**Comment by Phara0h** [comment]
@jason0x43 Do you have an estimate when this code can get merged good sir?

**Comment by jason0x43** [comment]
I'll review it this evening. From a quick glance it looks good, though.

File: stubs/homeassistant/const.pyi [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ATTR_ENTITY_ID: str
 ATTR_HIDDEN: str
 ATTR_TEMPERATURE: str
+ATTR_PRESSURE: str
```
Comment (jason0x43) [inline stubs/homeassistant/const.pyi:4]: There is no `ATTR_PRESSURE` constant in https://github.com/home-assistant/core/blob/dev/homeassistant/const.py (the stubs match up to the corresponding files in homeassistant/core).

**Review by jason0x43 — CHANGES_REQUESTED** [review]
This looks good! Just two requests:
1. Remove the invalid ATTR_PRESSURE constant declaration from the homeassistant/const stub
2. Alphabetize all the pressure-related additions (the constants, imports, and sensor class declarations are generally alphabetized)

File: stubs/homeassistant/const.pyi (hunk 1 above)
Comment (Phara0h) [reply stubs/homeassistant/const.pyi:4]: Thanks :D ill try to get this done today

**Comment by ottoyiu** [comment]
@Phara0h any updates on this PR? if you're busy, I can work on getting it ready.

**Comment by Phara0h** [comment]
Been super busy sorry, feel free to take it over :)

**Comment by ottoyiu** [comment]
@Phara0h thanks for the quick reply! I'm working to see if there's anyway the sensor can report the proper units:
https://github.com/home-assistant/core/blob/df25feab375639decd035d22221376bb86b0f04a/homeassistant/const.py#L426-L431

@jason0x43 I'm confused as to what needs to be done at first glance, it looks like @Phara0h's code follows the conventions of other classes, constants and imports.

```
Alphabetize all the pressure-related additions (the constants, imports, and sensor class declarations are generally alphabetized)
```

**Comment by jason0x43** [comment]
Hi, @ottoyiu! Regarding the alphabetization, values in the source files (constants, classes, etc) are generally in alphabetical order. Like, the list of all `ATTR_*` constants in const is alphabetized. That just makes it easier to scan the source when you're looking for something.

**Comment by ottoyiu** [comment]
@jason0x43 that's embarassing; I misread that as capitalization rather than alphabetization. Thanks for the clarification, I'll create a new PR with the changes. 

Thanks folks!

**Comment by ottoyiu** [comment]
Created #81 since I can't update the branch.

**Comment by jason0x43** [comment]
Closing this in favor of #81

**Pull request closed without merging**